### PR TITLE
Upgrade notes for the forced-id change

### PR DESCRIPTION
**Pull request by michaelabuckley**
Upgrade notes for #4803

### CODE REVIEW

File: hapi-fhir-docs/src/main/resources/ca/uhn/hapi/fhir/changelog/6_10_0/upgrade.md [hunk 1]
```diff
@@ -1,3 +1,13 @@
+### Major Database Change - Zero Downtime
+
+This release makes performance changes to the database definition in a way that is incompatible with releases before 2023.02.
+If you wish to upgrade with zero-downtime, you must first upgrade to 2023.02, or 2023.05.
```
Comment (fil512) [inline hapi-fhir-docs/src/main/resources/ca/uhn/hapi/fhir/changelog/6_10_0/upgrade.md:4]:
```suggestion
If you wish to upgrade with zero-downtime, you must first upgrade to 2023.02, or 2023.05 before upgrading to 2023.11.
```

File: hapi-fhir-docs/src/main/resources/ca/uhn/hapi/fhir/changelog/6_10_0/upgrade.md [hunk 2]
```diff
@@ -1,3 +1,13 @@
...
+If you are running an offline upgrade, you can upgrade from any prior version.
+On large systems, the database migration may take substantial time.
+For a zero-downtime upgrade, we recommend running the migration via the [`smileutil ` command](../../smileutil/migrate_database.html)
+before upgrading the running servers.
```
Comment (fil512) [inline hapi-fhir-docs/src/main/resources/ca/uhn/hapi/fhir/changelog/6_10_0/upgrade.md:8]: Question: is it possible writes could fail on the old servers during this process?

**Review by fil512 — APPROVED** [review]

File: hapi-fhir-docs/src/main/resources/ca/uhn/hapi/fhir/changelog/6_10_0/upgrade.md (hunk 2 above)
Comment (michaelabuckley) [reply hapi-fhir-docs/src/main/resources/ca/uhn/hapi/fhir/changelog/6_10_0/upgrade.md:8]: Not if they are 2023.02 or newer.  Yes, if older.  The new `FHIR_ID` column is changed to non-null with this release.  That's the "incompatible" part.

Comment (fil512) [reply hapi-fhir-docs/src/main/resources/ca/uhn/hapi/fhir/changelog/6_10_0/upgrade.md:8]: I guess it's not clear what exactly I need to do step by step to successfully migrate from older versions.

Maybe it is soemthing like this?
1. run cb migration up to X
2. upgrade all the servers to X and restart them
3. run db migration up to Y
4. upgrade all the servers to Y and restart them

Comment (michaelabuckley) [reply hapi-fhir-docs/src/main/resources/ca/uhn/hapi/fhir/changelog/6_10_0/upgrade.md:8]: We already have those docs - https://smilecdr.com/docs/installation/upgrading.html#upgrading-a-cluster-of-servers-with-zero-downtime

I'll link to them.

**Pull request merged**